### DISCUSSION TITLE
FAQ Series: Arduino usage

### DISCUSSION THREAD

**Discussion by Pavljava** [Electrostat-Lab/AVR-Sandbox#74] [Q&A] [answered]
Can we use arduino to compile and build those examples ?

**Comment by pavly-gerges** [comment, marked as answer]
Yes, since arduino uses atmega then you could still experience and test those examples on arduino, but you will need the PORTS datasheet linked in the readme.

FYI: 
1) Arduino UNO -> uses -> atmega32 chip.
2) Arduino NANO -> uses -> atmega328p chip.

**Comment by pavly-gerges** [comment]
More about datasheets: 
- [Atmega32A-Datasheet](https://github.com/Software-Hardware-Codesign/AVR-Sandbox/blob/master/MCU-Datasheets/Atmega32A-DataSheet-Complete-DS40002072A.pdf)
- [Arduino-UNO Datasheet](https://github.com/Software-Hardware-Codesign/AVR-Sandbox/blob/master/MCU-Datasheets/A000066-datasheet.pdf)
- [Atmega328p-Datasheet](https://github.com/Software-Hardware-Codesign/AVR-Sandbox/blob/master/MCU-Datasheets/Atmel-7810-Automotive-Microcontrollers-ATmega328P_Datasheet.pdf)
- [Arduino-NANO Datasheet](https://github.com/Software-Hardware-Codesign/AVR-Sandbox/blob/master/MCU-Datasheets/ArduinoNanoManual23.pdf)
- Atiny
